internal/resources/audit_log_export_schema.go

```diff
@@ -7,16 +7,16 @@ import (
 func AuditLogExportSchema() schema.Schema {
 	return schema.Schema{
 		Attributes: map[string]schema.Attribute{
-			"id":                     stringAttribute([]string{required}),
+			"id":                     stringAttribute([]string{computed}),
```

Comment:
Here this  field, i guess it should also be useStateForUnknown

Reply:
Addressed in fa68ea549a2155dd33a57deb780b9bde69b50129

Reply:
@aniket-Kumar-c when I run `terraform apply` after making this change I get this error:

```
╷
│ Error: Failed to load plugin schemas
│
│ Error while loading schemas for plugin components: Failed to obtain provider schema: Could not load the schema for provider registry.terraform.io/couchbasecloud/couchbase-capella: failed to retrieve
│ schema from provider "registry.terraform.io/couchbasecloud/couchbase-capella": Error converting resource schema: The schema for the resource "couchbase-capella_audit_log_export" couldn't be converted
│ into a usable type. This is always a problem with the provider. Please report the following to the provider developer:
│
│ AttributeName("id"): must have Required, Optional, or Computed set..
```

Not sure why `id` must be `required`, `optional` or `computed` ?

Reply:
Addressed in 3cbe8f5181d561b66ee1a1cb95aea6ac0363bb8c